pkgs/jni/pubspec.yaml

```diff
@@ -30,7 +30,7 @@ dependencies:
 dev_dependencies:
   dart_flutter_team_lints: ^3.2.0
   ## Pin ffigen version because we are depending on internal APIs.
```

Comment:
I believe we should run `dart analyze` on the JNIgen code-base in the FFIgen workflow to catch someone forgetting to migrate JNIgen.

Reply:
I could even run the generation.